boot/x86_64/setup.cc

```diff
@@ -52,18 +52,25 @@ extern "C" void _systemSetup(BootInfo *info) {
     "or $0x00000600, %eax\n\t"  // enable SSE and SSE exceptions
     "mov %eax, %cr4\n\t");      // the OSFXSR and OSXMMEXCPT bits
 
+  if (info->numMemoryRegions == 0 ||
```

Comment:
Why this formating?

Reply:
Yeah this line was longer initially, it can fit in one.